gap/callwithtimeout.gd

```diff
@@ -0,0 +1,51 @@
+#############################################################################
+##
+#F  IO_CallWithTimeout( <timeout>, <func>[, <arg1>[, <arg2>....]] )  
+##         . . call a function with a time limit
+#F  IO_CallWithTimeoutList( <timeout>, <func>, <arglist> )  
+##
+##  <#GAPDoc Label="IO_CallWithTimeout">
```

Comment:
This function won't appear in the manual unless you `<#Include` this chunk somewhere.

Reply:
woops, fixed